internal/clientconn/conn.go

```diff
@@ -140,7 +146,27 @@ func (c *conn) run(ctx context.Context) (err error) {
 	}()
 
 	bufr := bufio.NewReader(c.netConn)
+
+	// if test record path is set, split netConn reader to write to file and bufr
+	if c.testRecordPath != "" {
+		if err := os.MkdirAll(c.testRecordPath, 0o644); err != nil {
+			return err
+		}
+
+		path := filepath.Join(c.testRecordPath, "record.log")
```

Comment:
We should create different file names for every client (one for connection). Right now we will save all data in `record.log`. We can use `c.netConn` to get client remote addr and add current time to create sort of unique file name.

Reply:
Nice catch, forgot about that :smile: